### PR TITLE
Generate server on running init

### PR DESCRIPTION
**Pull request by creativej**
Update the `init` command to also generate a runnable server to help getting started even easier.

### CODE REVIEW

File: codegen/templates/server.gotpl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+{{- range $import := .Imports }}
+	{{- $import.Write }}
+{{ end }}
+)
+
+func main() {
+	port := flag.Int("p", 8080, "port number")
```
Comment (vektah) [inline codegen/templates/server.gotpl]: the PORT env var is commonly used for this (and removes the flag dep)

File: codegen/build.go [hunk 2]
```diff
@@ -99,6 +106,21 @@ func (cfg *Config) resolver() (*ResolverBuild, error) {
 	}, nil
 }
 
+func (cfg *Config) server(destDir string) *ServerBuild {
+	imports := Imports{
+		destDir: destDir,
+	}
+	imports.add(cfg.Exec.ImportPath())
```
Comment (vektah) [inline codegen/build.go]: you probably should include any imports that aren't covered by https://github.com/vektah/gqlgen/blob/master/codegen/import_build.go#L11-L27 here too (`http` and `githhub.com/vektah/gqlgen/handler`)

Without them, gqlgen can end up with duplicate package aliases in the generated code.

Comment (creativej) [reply codegen/build.go]: hrmm I didn't encounter that issue but sure. Updated.

**Comment by ermik** [comment]
I am not getting a server on bare `init` run nor does it appear with `--server server.go`. Maybe I am doing it wrong?

**Comment by ermik** [comment]
Aha! It works in a non-empty directory, but it only creates the yaml. Got it.

**Pull request merged**